### DISCUSSION TITLE
Reasoning the default is for "Enable JavaScript aspect ratio controls" to be off?

### DISCUSSION THREAD

**Discussion by CCpt5** [thomasasfk/sd-webui-aspect-ratio-helper#28] [Announcements]
Hi, 

First off thanks a lot for developing this - I've been hoping for a way to link the height/width sliders (lock them) as your extension does.  
  
That said, I'm curious why the default after installing is for the button next to the sliders, "Enable JavaScript aspect ratio controls", to be unchecked/off? To be honest, I downloaded your extension a few days ago hoping to see something like that, but didn't notice/pick up that there was an option in settings to add a button next to the sliders - so I uninstalled. Fortunately, I saw there were a few commits recently so I double checked and this time watched the video on the front page. That's the feature you demo there, and personally I think the most interesting/useful feature - the little lock/pull down.  
  
Just curious if there was a reason the default is off. Maybe it was added after launch? Also, I personally think naming the settings option to add the pulldown next to the sliders "Enable JavaScript aspect ratio controls" might not get the point across that it'll add a lock button next to the sliders up top. Just my though on it as a non-developer. 

Thanks again!!

**Comment by thomasasfk** [comment]
Hey! Sorry I've just seen this, didn't know about the discussion section... 

I completely agree with you, I think we'll change the default for the JavaScript controls and not the accordion.  

The reason I didn't do that initially, is that originally we didn't have the JavaScript controls, and I didn't want to surprise existing users with a feature randomly, so figured it'd be best to make it optional.   

Thanks for mentioning this, I'll try and do it after work tomorrow.

**Comment by thomasasfk** [comment]
I've actually just done this quickly since it's easy to do, the PR is here https://github.com/thomasasfk/sd-webui-aspect-ratio-helper/pull/35

As for the labels, I agree JavaScript controls is not the best naming convention.

If you'd like, you can suggest alternatives or raise a pull request, it should be relatively simple to change since they're just strings: 
https://github.com/thomasasfk/sd-webui-aspect-ratio-helper/blob/36c450111f44b059df5247f5d3dac4efe7cffb71/aspect_ratio_helper/_settings.py#L92-L112
